packages/nextjs/src/vendor/crypto-js.js

```diff
@@ -0,0 +1,5 @@
+import { AES } from 'crypto-es/lib/aes';
```

Comment:
Should we rename this file to `crypt-es.js` ?

Reply:
Yeah, good call